src/engine.js

```diff
@@ -177,7 +207,7 @@ Engine.prototype._onReloadSuccess = function _onReloadSuccess (serverResponse, s
 Engine.prototype._onReloadError = function _onReloadError (serverResponse) {
   var errors = serverResponse;
   this._modelUpdater.setErrors(errors);
```

Comment:
@ivan `this._modelUpdater.setErrors` expects instances of [WindshaftError](https://github.com/CartoDB/cartodb.js/blob/1819-use-engine/src/windshaft/error.js).

This is how it was done before: https://github.com/CartoDB/cartodb.js/blob/v4/src/windshaft/map-base.js#L59-L60

We probably need a test for this... 😉